platform/broadcom/sonic-platform-modules-inventec/common/utils/led_proc.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2018 Inventec, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
```

Comment:
can you remove license from the file.

Reply:
sure. done in commit 4ddf83d84a84f583843948c01fc581c6ebcb0c71